lib/main.dart

```diff
@@ -37,5 +37,10 @@ Future<void> main() async {
     await androidDisplayInitialization(widgetsBinding);
   }
 
-  runApp(ProviderScope(observers: [ProviderLogger()], child: const AppInitializationScreen()));
+  runApp(
```

Comment:
You have a problem with your formatter. It should not reformat the files. It should be set to 100.